### PR TITLE
Add pending-tests indicator to tab label

### PR DESCRIPTION
**Pull request by karthikbtf**
- also show seed bank name instead of storage location name

Terraware App 2022-09-20 10-10-55

### CODE REVIEW

File: src/components/accession2/view/Accession2View.tsx [hunk 1]
```diff
@@ -343,11 +373,11 @@ export default function Accession2View(props: Accession2ViewProps): JSX.Element
             </Box>
           </Box>
         )}
-        {accession?.storageLocation && (
+        {accession?.facilityId && (
           <Box sx={editableParentProps}>
             <Icon name='iconMyLocation' className={classes.iconStyle} />
             <Box sx={editableProps}>
-              <Typography paddingLeft={1}>{accession.storageLocation}</Typography>
+              <Typography paddingLeft={1}>{getSeedBank(organization, accession.facilityId)?.name}</Typography>
```
Comment (constanzauanini) [inline src/components/accession2/view/Accession2View.tsx:380]: We are showing the seed bank only now, and not the storage location. Is that what we want?

Comment (karthikbtf) [reply src/components/accession2/view/Accession2View.tsx:380]: i can change this to `seed bank name (italic storage location name)` and we can iterate  on that?

Comment (karthikbtf) [reply src/components/accession2/view/Accession2View.tsx]: Added storage location, we can change the display later if Ming has another suggestion.

Terraware App 2022-09-20 10-27-29

Comment (karthikbtf) [reply src/components/accession2/view/Accession2View.tsx]: Ming commented that we need to show both.

Comment (constanzauanini) [reply src/components/accession2/view/Accession2View.tsx]: Thanks!

**Review by constanzauanini — APPROVED** [review]

File: src/components/accession2/view/Accession2View.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/accession2/view/Accession2View.tsx]: I asked if my current proposal "seed bank name (storage location name)" is ok. if she suggests something else I will send a new PR

**Pull request merged**